api/project/v1/project.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package clutch.project.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/project/v1;projectv1";
+
+import "api/v1/annotations.proto";
+import "core/project/v1/project.proto";
+import "google/api/annotations.proto";
+import "validate/validate.proto";
+
+service ProjectAPI {
+  rpc GetProjects(GetProjectsRequest) returns (GetProjectsResponse) {
+    option (google.api.http) = {
+      post : "/v1/project/getProjects"
+      body : "*"
+    };
+    option (clutch.api.v1.action).type = READ;
+  }
+}
+
+message GetProjectsRequest {
+  repeated string names = 1 [ (validate.rules).repeated = {min_items : 1} ];
+}
+
+message GetProjectsResponse {
+  repeated clutch.core.project.v1.Project projects = 1;
```

Comment:
we may want to preemptively wrap each result in a Message in case we want to somehow extend the result response without adding fields to project, kind of like how we ended up in a place where attaching metadata in the resolver API would result in a breaking change or a less optimal API. given that it's early we can also just go with this and reconsider before some initial usage of the API.

Reply:
Yeah good idea, let me leave a TODO.